README.md

```diff
@@ -66,6 +66,13 @@ At this point, `hauser` will perform the process approximately every `ExportDura
 When using a database, it uses the `SyncTable` to keep track of what export files have been processed, and will restart from the last known sync point.
 For a `StorageOnly` process, it will create a file called `.sync.hauser` that will be used as a checkpoint.
 
+### Syncing Specific Segments
+_Note: this field is optional and should only be set if syncing a specific segment._
+
+By default `hauser` will sync the "Everyone" segment. If you would rather sync a specific segment, you can specify a value by setting the `SegmentId` config option. You can use the [list segment endpoint](https://developer.fullstory.com/server/v1/segments/list-segments/) to get a list segment endpoint to get a list of IDs of segments in your org.
+
+Alternatively, the segment's URL in FullStory will contain the segment's ID. For example, a segment with the following URL `https://app.fullstory.com/ui/ABCD/segments/alice@example.com:12444435241/people/0` has a segment ID of `alice@example.com:12444435241`. You would then add `SegmentId = "alice@example.com:12444435241"` to your toml.
```

Comment:
Oof, I know that the dev docs use the email prefix, but this is the old version of our segment Ids. This has since been updated. Maybe change this to be something more random or state that older segment IDs may look like `X`?

Reply:
Sorry Cam. I used this page: https://developer.fullstory.com/server/v1/segments/list-segments/ for the examples. Didn't realize it was v1. It's also hard to find the segment export stuff now on the dev docs. Looks like anything not v2 isn't shown in the nav bar (screenshot). 

Is the segment id highlighted in [this kb doc](https://help.fullstory.com/hc/en-us/articles/360057606414-HTTP-API-Using-Segment-Export#01ER5MMG4E8FZNTNB98XPD49JJ) correct? 

![image](https://github.com/fullstorydev/hauser/assets/5333258/3c02ed31-c0fa-4f11-84da-304084a2df72)